tests/scripts/create-dev-cluster.sh

```diff
@@ -68,11 +75,16 @@ show_ceph_dashboard_info() {
 check_minikube_exists() {
     minikube profile list > /dev/null 2>&1
     local retcode=$?
-
-    if [ $retcode -eq 0 ]; then
-        echo "A minikube environment already exists, please use -f to force the cluster creation."
-	exit 1
-    fi
+    	if [ $retcode -eq 0 ]; then
+		echo "A minikube environment already exists."
+		echo "Using the existing minikube environment."
+		echo "       please use -f to force the minikube cluster creation."
+		return  1
+	else
+		echo " A minikube environment does not exist yet."
+		echo "       please use -f to force the minikube cluster creation."
+    		return 0
+	fi
```

Comment:
I have another PR where I handle this by using minikube profiles this way we avoid the confusion I think.

Reply:
@rkachach  wrote:
> I have another PR (#13158) where I handle this by using minikube profiles this way we avoid the confusion I think.

Thanks @rkachach ! This makes a lot of sense! I updated this PR to remove the minikube related commit and prepared a branch for your PR rebased on top of this one: 

https://github.com/obnoxxx/rook/commits/pr-13158-rebased-on-pr-13155